java/connector-node/risingwave-source-cdc/src/main/java/io/debezium/connector/postgresql/connection/pgoutput/PgOutputMessageDecoder.java

```diff
@@ -363,6 +363,10 @@ private Table handleRelationMessage(ByteBuffer buffer, TypeRegistry typeRegistry
         columnDefaults =
                 readColumns.stream()
                         .filter(io.debezium.relational.Column::hasDefaultValue)
+                        .filter(
+                                column ->
+                                        isConstantDefaultValue(
```

Comment:
Filtering out non-const default here can be dangerous because it also affects the debezium internal [decoder state](https://github.com/debezium/debezium/blob/4159397bd40dd5d0c8efe2b97ba61ba8eb220b58/debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/pgoutput/PgOutputMessageDecoder.java#L363). Given that it is RW that doesn't care about the non-const default value rater than debezium, it is better we do it in the RW's code instead of inside debezium.